### ISSUE TITLE
[Tracking]: Combine devmode and prodmode into ESM 2020

### ISSUE THREAD

**Issue by devversion** [angular/dev-infra#365]
Notes:

* How will it work with ZoneJS if we target ES2020? custom transform?

**Comment by alan-agius4** [comment]
> * How will it work with ZoneJS if we target ES2020? custom transform?

Or babel like we do in the Angular CLI.

**Comment by devversion** [comment]
Yeah, need to be careful though as this is super slow based in my recent testing. Potentially ESBuild could do the job?
